test/test_jit_cuda_fuser.py

```diff
@@ -513,7 +513,7 @@ def t(x: torch.Tensor, y: torch.Tensor, z: float):
         self.assertEqual(o, jit_o)
         self.assertGraphContains(t_jit.graph_for(x, y, 2.0), FUSION_GROUP)
 
-    @unittest.skipIf(not RUN_CUDA, "requires CUDA")
+    @unittest.skip("temp disable while switching to Profiling Executor")
```

Comment:
Nit: we probably want to still keep `@unittest.skipIf(not RUN_CUDA, "requires CUDA")`. AFAIK, these decorators can be combined.

Reply:
a good point! updated the PR